library/core/src/intrinsics.rs

```diff
@@ -1787,6 +1787,8 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
+    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
```

Comment:
Somewhat ambiguous what "this" in "this is not something that can actually be used" is. Is it this intrinsic itself? Or the very fact that the exceptions might be raised and you cannot utilize this fact for anything.

Reply:
It was clear to me that "this" meant the raised exception, but we can always tweak it further!